### PR TITLE
Ktlint configuration

### PR DESCRIPTION
**Pull request by florenciaferraiolo**
## Description

Added Ktlint configuration in build.gradle

## Evidences

Captura de Pantalla 2022-05-16 a la(s) 13 38 36

Captura de Pantalla 2022-05-16 a la(s) 13 38 53

Captura de Pantalla 2022-05-16 a la(s) 13 39 03

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -33,11 +37,32 @@ android {
...
+}
+check.dependsOn ktlint
+
+task ktlintFormat(type: JavaExec, group: "formatting") {
+    description = "Fix Kotlin code style deviations."
+    classpath = configurations.ktlint
+    main = "com.pinterest.ktlint.Main"
+    args "-F", "src/**/*.kt"
+}
+preBuild.dependsOn ktlintFormat
```
Comment (SartiMau) [inline app/build.gradle]: to avoid this error, add an empty line at the end of the file

File: app/build.gradle [hunk 2]
```diff
@@ -33,11 +37,32 @@ android {
...
-    implementation 'com.google.android.material:material:1.5.0'
+    implementation 'com.google.android.material:material:1.6.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.1.3'
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
-}
\ No newline at end of file
+}
+java {
```
Comment (SartiMau) [inline app/build.gradle:44]: all the ktlint related things should be in a separated file

File: .idea/compiler.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (SartiMau) [inline .idea/compiler.xml:1]: please, add this file to the gitIgnore

File: .idea/misc.xml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (SartiMau) [inline .idea/misc.xml:1]: please, add this file to the gitIgnore

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 1 above)
Comment (florenciaferraiolo) [reply app/build.gradle]: done

File: app/build.gradle (hunk 2 above)
Comment (florenciaferraiolo) [reply app/build.gradle:44]: done

File: .idea/compiler.xml (hunk 3 above)
Comment (florenciaferraiolo) [reply .idea/compiler.xml:1]: done

File: .idea/misc.xml (hunk 4 above)
Comment (florenciaferraiolo) [reply .idea/misc.xml:1]: done

**Review by SartiMau — APPROVED** [review]

**Review by Georgina-Fernandez — APPROVED** [review]

**Review by mariano98montero — APPROVED** [review]

**Pull request merged**